### PR TITLE
Update MotionTransitioning dependency to 4.0.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
Will send the CocoaPods file updates in a separate PR.

File: demos/Bare/Podfile.lock [hunk 1]
```diff
@@ -341,13 +341,13 @@ EXTERNAL SOURCES:
...
   MotionAnimator: 5a1893b58fcf7a4664d81d69c9a09a0078c67e06
-  MotionInterchange: b600100244df56c4f462bdacbcd11c44e6305d05
-  MotionTransitioning: caaa488e0469d93f004793b96a2ed04447af808d
+  MotionInterchange: 3556642f403549449a49f05653da3bd932a5e072
+  MotionTransitioning: be4161ebcbff7911a1d9c4549e396f486041ca6f
 
 PODFILE CHECKSUM: 8c6ad4fe8e5aa667e06c9b43f3f5d4d6430b1322
 
-COCOAPODS: 1.2.1
+COCOAPODS: 1.3.1
```
Comment (romoore) [inline demos/Bare/Podfile.lock]: Should I update my cocoapods version?

**Review by romoore — APPROVED** [review]

**Comment by jverkoey** [comment]
Yeah this PR makes it seem like we should all probably update.

**Comment by jverkoey** [comment]
lock files being updated in #2257.

**Comment by ianegordon** [comment]
Tested locally on iPhone X Sim.
Please merge when Travis is happy.

**Review by ianegordon — APPROVED** [review]

**Pull request merged**